index.js

```diff
@@ -20,6 +22,109 @@ app.get('/info', (req, res) => {
   res.send({ serverName: packageData.name, serverVersion: packageData.version });
 });
 
+// route for displaying all products
+app.get('/products/all', (req, res) => {
+
+  var i, j;
+  var jsonOutputObject;
+
+  // Constructing JSON which lists all the products category wise
+  jsonOutputObject = '{"product_list_categorically": [';
```

Comment:
Do we actually need to create this as a string JSON?

Reply:
Sorry, I'm actually new to JS, so I have been making a lot of mistakes. Anyway I've resolved this issue by using a JSON object directly :)

Thanks for reviewing, learning a lot of new things with these tasks ;)